docs/src/app/components/pages/components/BottomNavigation/ExampleSimple.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import FontIcon from 'material-ui/FontIcon';
+import {BottomNavigation, BottomNavigationItem} from 'material-ui/BottomNavigation';
+import Paper from 'material-ui/Paper';
+
+const recentsIcon = <FontIcon className="material-icons">restore</FontIcon>;
+const favoritesIcon = <FontIcon className="material-icons">favorite</FontIcon>;
+const nearbyIcon = <FontIcon className="material-icons">location_on</FontIcon>;
+
+class BottomNavigationExampleSimple extends React.Component {
```

Comment:
1. Direct import: `extends Component {`
2. Arrow function binding: `select = (index) => this.setState({selectedIndex: index});`
3. Class properties: `state = {selectedIndex: 0}`